crates/rnote-ui/src/appwindow/actions.rs

```diff
@@ -782,19 +813,20 @@ impl RnAppWindow {
 
         // shortcuts for devel build
         if config::PROFILE.to_lowercase().as_str() == "devel" {
-            app.set_accels_for_action("win.visual-debug", &["<Ctrl><Shift>v"]);
+            app.set_accels_for_action("win.visual-debug", &["<Ctrl><Alt>v"]); //clashes with the special paste method
```

Comment:
I would like to use `<Ctrl><Shift>v` for the second paste method that respect borders, should I leave this shortcut intact and change the one for the paste method or change this one (not sure this is the best keyboard shortcut to change the visual debug call to).

Reply:
It's fine to me to change the visual-debug shortcut. To be honest it can be any shortcut, it's just sometimes convenient to toggle it via keyboard for quick debugging. But not important which actual keys need to be pressed.

Reply:
Ok then.
I think this PR is done, I want to do the other resizing part in another PR and stop at changing only the image import here.

The one thing that kinda works but could be done better is the drag and drop part. I changed things so that `shift + drag and drop` activate the other mode but that only works if the window is in focus to begin with. If you know how to do it correcly I can change the related code, I can also keep it as is, or remove it entirely (It's not the most beautiful thing ever ..)